### PR TITLE
Fix testcase TC-DRLK-2.9 

### LINKED ISSUE #21644: Fix and enable YAML for TC-DRLK-2.9 to align with the test plan
Please make sure there is a follow-up to align this YAML with the test plan, at least, and re-enable it here.

_Originally posted by @bzbarsky-apple in https://github.com/project-chip/connectedhomeip/pull/21319#discussion_r936227725_

### PR DESCRIPTION
**Pull request by manjunath-grl**
Fixed DRLK-2.9
Fixes issue [21644](https://github.com/project-chip/connectedhomeip/issues/21644)
Added auto generated files

Tested
Attached execution logs for reference:
[Execution_log.txt](https://github.com/project-chip/connectedhomeip/files/9267037/Execution_log.txt)

### CODE REVIEW

**Review by jrhees-cae — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_DRLK_2_9.yaml [hunk 1]
```diff
@@ -90,9 +90,8 @@ tests:
           saveAs: NumberOfTotalUsersSupported
           constraints:
               minValue: 0
-              maxValue: 255
+              maxValue: 65534
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_DRLK_2_9.yaml:93]: Why is this 65534?  I don't see that constraint in the spec.

File: src/app/tests/suites/certification/Test_TC_DRLK_2_9.yaml [hunk 2]
```diff
@@ -81,7 +81,7 @@ tests:
                 value: null
 
     - label:
-          "TH reads NumberOfTotalUsersSupported attribute and saves for future
+          "TH reads NumberOfTotalUsers Supported attribute and saves for future
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_DRLK_2_9.yaml:84]: Why this change?  The attribute is called NumberOfTotalUsersSupported.

Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_DRLK_2_9.yaml:84]: my mistake i will fix this

File: src/app/tests/suites/certification/Test_TC_DRLK_2_9.yaml (hunk 1 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_DRLK_2_9.yaml:93]: @bzbarsky-apple it is mentioned in testplan: valid number in the inclusive range of 0 to 65534
image

**Comment by manjunath-grl** [comment]
We had 3 issue(20991 , 20930 and 19990)  opened for TC-DRLK-2.9 we verified issue 20991 and 20930 and script is updated in PR , however issue reported in 19990  still seems to be present( expects response of 0X03 (OCCUPIED), but recevied a response of 0X02(Duplicate) ) QA team also validated same response.
https://docs.google.com/spreadsheets/d/1h4d4J90KAxFqUL_cXZHyM3pHjHCEXzFITkQhN2KMqDY/edit#gid=1936774884
